lsvmutils/efiblkdev.c

```diff
@@ -77,15 +77,11 @@ static int _Close(
 }
 
 static int _Get(
-    Blkdev* dev,
+    BlkdevImpl* impl,
     UINTN blkno,
     void* data)
 {
     int rc = -1;
-    BlkdevImpl* impl = (BlkdevImpl*)dev;
-
-    if (!impl || !data || !impl->bio)
-        goto done;
```

Comment:
I assume this was already checked by the caller?

Reply:
Yeah, its an internal function now. Only called by GetN, which checks the parameters.